src/cms/pages-search.md

```diff
@@ -40,6 +40,18 @@ _Filtered search_
 |Store View|Filter the search based on store view. Options: All available / Store Views|
 |Layout|Filter the search based on page layout. Options: 1 column / 2 columns with left bar / 2 columns with right bar / 3 columns / Empty|
 |Status|Filter the search on the page status. Options: Disables / Published|
-|Apply Filters|Applies all filters to the search.|
+|Custom design from / to|Filter the search by the start and end date when the selected design is applied to the product page|
+|Asset|Filter the search by the page title assets|
+|Custom Layout|Filter the search based on custom layout. Options: 1 column / 2 columns with left bar / 2 columns with right bar / 3 columns / Empty / Page -- Full Width / Category -- Full Width / Product -- Full Width|
```

Comment:
This table looks strange with the slash between options. I know you didn't start that, but can we use break tags to list the options in this table?

Reply:
This is how we have typically done it when there is a discrete list of options available. We only do a <br/> if we are providing a description of each option.